nixos/modules/services/mail/postsrsd.nix

```diff
@@ -7,6 +7,19 @@
 let
 
   cfg = config.services.postsrsd;
+  runtimeDirectoryName = "postsrsd";
```

Comment:
```suggestion
```

Reply:
I'd like to keep this near the `runtimeDirectory` variable on the next line. Instead, I've updated `runtimeDirectory` to directly reference this variable.

I'm going to mark this (and the related comment below) resolved. Feel free to reopen if you disagree.